src/protocol-parser/protocol-parser.ts

```diff
@@ -310,6 +310,7 @@ export namespace Script {
     CommonDataTypes.RemoteReferenceSchema,
     CommonDataTypes.SharedReferenceSchema,
     CommonDataTypes.LocalValueSchema,
+    Script.ChannelSchema,
```

Comment:
`Script.ChannelSchema` is not set at this point. Either move `Script.ChannelSchema` above this declaration, or use `zod.lazy(() => ...`.

Reply:
Done